src/svindexer.ts

```diff
@@ -117,6 +118,27 @@ export class SystemVerilogIndexer {
         this._userDefines = defines.map(s => s.split("=", 2)).map(s => [s[0], s[1], s[1] == undefined ? [] : SystemVerilogPreprocessor.tokenize(s[1])]);
     }
 
+    setLibraries(libraries: string[], excludes: string[]) {
+        let _indexGlob = (func) => {
+            const pattern = libraries.length == 1 ? libraries[0] : '{' + libraries.join(",") + '}';
+            glob(pattern, {cwd: this._rootPath, ignore: excludes, follow: true, realpath: true}, func);
+        };
+        _indexGlob((err, files) => {
+            if (err) {
+                ConnectionLogger.error(err);
+            }
+            else if (files.length > 0) {
+                if (!genutils_1.isStringListEqual(files, this._libFiles)) {
+                    this._libFiles = files;
+                }
+            }
+            else {
+                ConnectionLogger.log("Library file is not found");
```

Comment:
Right now this is worded such that it might look like an error. Would request the wording be changed to "No library files found"

Reply:
Agree. The wording is changed.